lib/manager/cargo/artifacts.spec.ts

```diff
@@ -65,11 +59,13 @@ describe('manager/cargo/artifacts', () => {
       })
     ).toBeNull();
   });
+
   it('returns null if unchanged', async () => {
     fs.stat.mockResolvedValueOnce({ name: 'Cargo.lock' } as any);
-    fs.readFile.mockResolvedValueOnce('Current Cargo.lock' as any);
+    fs.findLocalSiblingOrParent.mockResolvedValueOnce('Cargo.lock' as any);
+    fs.readLocalFile.mockResolvedValueOnce('Current Cargo.lock' as any);
     const execSnapshots = mockExecAll(exec);
-    fs.readFile.mockResolvedValueOnce('Current Cargo.lock' as any);
+    fs.readLocalFile.mockResolvedValueOnce('Current Cargo.lock' as any);
```

Comment:
```suggestion
    fs.readLocalFile.mockResolvedValueOnce('Current Cargo.lock');
```
Please remove those `any` casts, should work without.